common/app/assets/assets.scala

```diff
@@ -1,19 +1,24 @@
 package common.Assets
 
+import java.net.URL
+
 import common.{Logging, RelativePathEscaper}
 import conf.Configuration
 import model.ApplicationContext
 import org.apache.commons.io.IOUtils
 import play.api.libs.json._
 import play.api.Mode
 
-import scala.collection.concurrent.{Map => ConcurrentMap, TrieMap}
+import scala.collection.concurrent.{TrieMap, Map => ConcurrentMap}
 import scala.util.{Failure, Success, Try}
 
 // turns an unhashed name into a name that's hashed if it needs to be
 class Assets(base: String, mapResource: String, useHashedBundles: Boolean = Configuration.assets.useHashedBundles) extends Logging {
 
-  lazy val lookup: Map[String, String] = Get(assetMap(mapResource))
+  lazy val lookup: Map[String, String] = assetMap(mapResource) match {
+    case Success(s) => s
+    case Failure(e) => throw e
```

Comment:
Again can we log an error and instantiate an empty map here rather than throwing an exception?